### PR TITLE
PIM-4980: remove formatter from localization component

### PR DESCRIPTION
**Pull request by skeleton**
| Q                 | A
| ----------------- | ---
| Specs             | N
| Behats            | N
| Blue CI           | :white_check_mark: 
| Changelog updated | N
| Review and 2 GTM  | :white_check_mark: 
| Micro Demo (PO)   | N
| Migration script  | N
| Tech Doc          | N

### CODE REVIEW

File: src/Pim/Component/Localization/Localizer/AbstractNumberLocalizer.php [hunk 1]
```diff
@@ -44,6 +42,22 @@ public function convertDefaultToLocalized($number, array $options = [])
...
+     * {@inheritdoc}
+     */
+    public function convertDefaultToLocalizedFromLocale($number, $locale)
+    {
+        if (null === $number || ''  === $number) {
+            return $number;
+        }
+
+        $numberFormatter = new \NumberFormatter($locale, \NumberFormatter::DECIMAL);
+        $numberFormatter->setAttribute(\NumberFormatter::MIN_FRACTION_DIGITS, 2);
```
Comment (pierallard) [inline src/Pim/Component/Localization/Localizer/AbstractNumberLocalizer.php]: Does it work for integers ? (non decimal-allowed)

**Comment by pierallard** [comment]
Tiny remark then GTM

File: src/Pim/Component/Localization/Localizer/LocalizerInterface.php [hunk 2]
```diff
@@ -52,6 +54,16 @@ public function convertLocalizedToDefault($data, array $options = []);
     public function convertDefaultToLocalized($data, array $options = []);
 
     /**
+     * Convert a default value to a localized format depending on a locale (e.g. from UI)
+     *
+     * @param $data
```
Comment (solivier) [inline src/Pim/Component/Localization/Localizer/LocalizerInterface.php]: missing type

**Comment by solivier** [comment]
small comment then GTM

**Pull request merged**